ui/app/main.go

```diff
@@ -37,15 +37,14 @@ func main() {
 		ctx, cancel := context.WithCancel(context.Background())
 		defer cancel()
 
-		const elmMake = "elm-make"
-		elmMakeArgs := []string{"src/Main.elm", "--output", "script.js"}
+		elmMakeArgs := []string{"make", "src/Main.elm", "--yes", "--output", "script.js"}
```

Comment:
Sorry, I should've opened a separate PR for this change. If you want, I can still do this.